java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -157,6 +162,8 @@ public static Object[] transactionTestResult() {
             Map.of("timeout", "1000"),
             OK,
             "GLIDE", // echo
+            0L, // rpushx(listKey3, new String[] { "_" })
```

Comment:
Will you change these to push to existing lists later on?

Reply:
No, that could be messy